ISMessages/Classes/ISMessages.m

```diff
@@ -283,8 +302,11 @@ - (void)showInMain {
                          }];
         
         [currentAlertArray addObject:self];
-        
-        if (_duration > 0.5f) {
+
+        UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification, self.view);
```

Comment:
Oops, just noticed this was LayoutChanged, instead of ScreenChanged. ScreenChanged emits a helpful sound for the user, like with default UIAlerts.

@EddyVerbruggen want me to fix this in another PR?

Reply:
That's a good idea, thank you! I'm just about to test your PR 👍